### PR TITLE
Update a document and error messages

### PR DESCRIPTION
**Pull request by d-nakamichi**
It updates,

* Document: `ConnectingRealRobot.md`
* Error messages mainly about `krnx_PrimeRtcCompData`

### CODE REVIEW

File: khi_robot_control/src/khi_robot_krnx_driver.cpp [hunk 1]
```diff
@@ -607,19 +609,19 @@ bool KhiRobotKrnxDriver::writeData( const int cont_no, JointData joint )
...
+            for ( int jt = 0; jt < p_rb_tbl[cont_no]->arm_tbl[ano].jt_num; jt++ )
             {
-                if ( p_rb_tbl[cont_no]->arm_tbl[ano].jt_tbl[cnt].type == TYPE_LINE )
-                {
-                    diff = ( rtc_comp[cont_no][ano][cnt] - rtc_old_comp[cont_no][ano][cnt] )/KHI_KRNX_M2MM*(1e+9/robot_info[cont_no].period);
-                }
-                else
+                jt_pos = data.ang_ref[jt];
+                rtc_diff = ( rtc_comp[cont_no][ano][jt] - rtc_old_comp[cont_no][ano][jt] )*(1e+9/robot_info[cont_no].period);
+                if ( p_rb_tbl[cont_no]->arm_tbl[ano].jt_tbl[jt].type == TYPE_RAD )
```
Comment (matsui-hiro) [inline khi_robot_control/src/khi_robot_krnx_driver.cpp]: In above docs ,` |diff|float|joint command difference value[deg or mm] between period|`
I think it is need to change [m] to [mm].

Comment (matsui-hiro) [reply khi_robot_control/src/khi_robot_krnx_driver.cpp]: `rtc_comp`  is   already [mm] units in this line,is it?
I had been misunderstanding.

But ROS standard units is SI units.
SO these error messages had better to been also displayed in [rad] or [m].

Comment (d-nakamichi) [reply khi_robot_control/src/khi_robot_krnx_driver.cpp]: OK, I changed units about error messages.

**Comment by d-nakamichi** [comment]
For #17, I added modifications to `ConnectingRealRobot.md` about real-time configuration using `limits.conf`.

File: docs/ConnectingRealRobot.md [hunk 2]
```diff
@@ -1,62 +1,97 @@
...
+Make sure that the Ubuntu PC used for real-time control satisfies the following conditions.
+
+* The PC is using realtime kernel for Ubuntu 16.04.
+* The user has real-time permissions.
+  * (e.g.)Making a real-time group named `realtime`
+    1. Make a group and add a user
+
+       ```bash
+       sudo addgroup realtime
+       sudo usermod -aG $(whoami)
```
Comment (matsui-hiro) [inline docs/ConnectingRealRobot.md]: I think this line is `sudo usermod -aG realtime $(whoami)`.

Comment (d-nakamichi) [reply docs/ConnectingRealRobot.md]: Thanks, I fixed it.

**Pull request merged**